skymp5-server/cpp/server_guest_lib/AnimationSystem.h

```diff
@@ -28,7 +31,15 @@ class AnimationSystem
     MpActor* actor,
     std::chrono::steady_clock::time_point timePoint =
       std::chrono::steady_clock::now());
+  std::vector<uint32_t> GetWeaponKeywordFormIds(uint32_t baseId) const;
+  std::vector<std::string_view> GetWeaponKeywords(uint32_t baseId) const;
+  float ComputeWeaponStaminaModifier(uint32_t baseId) const;
+  void HandleAttackAnim(MpActor* actor, float defaultModifier = 0.f) const;
+
+private:
+  static std::unordered_map<std::string_view, float> s_weaponStaminaModifiers;
```

Comment:
std::string_view -> std::string
static -> non-static

Reply:
I made it static to properly initialize this map using server config. This map will be recreated [here](https://github.com/skyrim-multiplayer/skymp/blob/578dbcf6b30a87733c8d8fd7c55d9aba2e69ff7b/skymp5-server/cpp/server_guest_lib/ActionListener.cpp#L155), unless it is not static.

Reply:
Ok. I see. Please refactor AnimationSystem creation instead of using `static`, it should not be too painful